specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19_Preview/OperationalInsights.json

```diff
@@ -123,6 +123,78 @@
         }
       }
     },
+    "/{resourceUri}/query": {
```

Comment:
Should we call this `resourceId` instead of `resourceUri`. Portal uses `Resource ID` and that's the term we should use here.

![image](https://user-images.githubusercontent.com/51379715/172474160-44c936d6-d1a3-4e30-b73f-49186b919477.png)